### PR TITLE
Digisub confirm cancel V1

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
New screen to confirm that users want to cancel their subscription

## How can we measure success?
Test via storybook as URLs don't yet exist

## Images
image

image

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+	palette,
+	space,
+	textSans,
+} from '@guardian/source-foundations';
+import { Button, Stack } from '@guardian/source-react-components';
+import { Link } from 'react-router-dom';
+import { benefitsCss } from '@/client/components/mma/shared/benefits/BenefitsStyles';
+import { stackedButtonLayoutCss } from '@/client/styles/ButtonStyles';
+
+function GreyBulletpoints() {
```
Comment (andrewHEguardian) [inline client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx:14]:
```suggestion
function GreyBulletpoint() {
```
Maybe should be singular, for clarity?

File: client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+				css={css`
+					${headline.xsmall({ fontWeight: 'bold' })};
+					margin-top: 0;
+					margin-bottom: 0;
+					${from.tablet} {
+						${headline.medium({ fontWeight: 'bold' })};
+					}
+				`}
+			>
+				Losing you supporter extras
```
Comment (andrewHEguardian) [inline client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx:91]:
```suggestion
				Losing your supporter extras
```

File: client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+			>
+				Please confirm to cancel your digital subscription
+			</div>
+			<div css={stackedButtonLayoutCss}>
+				<Button
+					cssOverrides={css`
+						background-color: ${palette.news['400']};
+						justify-content: center;
+						:hover {
+							background-color: ${palette.news['200']};
```
Comment (andrewHEguardian) [inline client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx:117]: How do we know what colour to make it on hover?

File: client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+						}
+					`}
+				>
+					Cancel subscription
+				</Button>
+				<Link
```
Comment (andrewHEguardian) [inline client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx:123]: I think if we use this type of [source button](https://guardian.github.io/csnx/?path=/story/source-react-components_button--subdued-priority-default-theme) we don't need to include so many styles (obviously then it has to use the onClick with navigate)

**Review by andrewHEguardian — APPROVED** [review]
Just a minor typo but otherwise looks good 👍

File: client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx (hunk 3 above)
Comment (charleycampbell) [reply client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx:117]: I used the same cancel button css we have on the membership cancellation saves journey

File: client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx (hunk 4 above)
Comment (charleycampbell) [reply client/components/mma/cancel/cancellationSaves/digipack/ConfirmCancellation.tsx]: Yeah good shout!

**Review by andrewHEguardian — APPROVED** [review]

**Pull request merged**